.github/workflows/inductor-unittest.yml

```diff
@@ -0,0 +1,153 @@
+# Workflow: Inductor Unit Test
+# 1. runs unit tests for inductor.
+# 2. perfoms daily memory leak checks and reruns of disabled tests, scheduled at `29 8 * * *`.
+
+name: inductor-unittest
+on:
+  workflow_call:
+  schedule:
+    - cron: 29 8 * * * # about 1:29am PDT, for mem leak check and rerun disabled tests.
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
You might need to double check the concurrency group here to make sure that it is different than the one used by the inductor workflow to avoid the cancellation error.  GH shows that error in the workflow summary https://github.com/pytorch/pytorch/actions/runs/11616458101

Reply:
added `inductor-unittest` in the end

Reply:
add optional input for reuseable workflow to pass jobname from the parent workflow